src/app/shared/shared.module.ts

```diff
@@ -1,7 +1,7 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { SpinnerComponent } from './components/spinner/spinner.component';
-import { MaterialModule } from '../material.module';
+import { MaterialModule } from './material/material.module';
 import { ConfirmComponent } from './components/confirm/confirm.component';
 
 
```

Comment:
Good practice is to create constant in case to avoid duplication in declarations and exports
const SHARED_COMPONENTS = [SpinerComponent, ConfirmComponent]
declarations: [...SHARED_COMPONENTS]
exports: [...SHARED_COMPONENTS]

Reply:
done